cmd/subnetcmd/upgradecmd/import_upgrade.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package upgradecmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/spf13/cobra"
+)
+
+var upgradeBytesFilePath string
+
+const upgradeBytesFilePathKey = "upgrade-filepath"
+
+// avalanche subnet upgrade import
+func newUpgradeImportCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import [subnetName]",
+		Short: "Import the upgrade bytes file into the local environment",
+		Long:  `Import the upgrade bytes file into the local environment`,
+		RunE:  upgradeImportCmd,
+		Args:  cobra.ExactArgs(1),
+	}
+
+	cmd.Flags().StringVar(&upgradeBytesFilePath, upgradeBytesFilePathKey, "", "Import upgrade bytes file into local environment")
+	if err := cmd.MarkFlagRequired(upgradeBytesFilePathKey); err != nil {
```

Comment:
Instead of marking this as required, can we prompt for this? As part of our design philosophy, I don't think we want to ever make flags mandatory.

Reply:
Our design philosophy is also that we can run things through without having to prompt. Which is conflicting.

Reply:
Oh ok, keep the flag but don't make it required. Got it.